### PR TITLE
 plugins:ansible - Release 0.2.1 [deb]

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

**Comment by ares** [comment]
Scratch build [failed](http://ci.theforeman.org/job/packaging_build_deb_plugin/arch=x86,label=debian/880/console) with

```
dh clean
   dh_testdir
   dh_auto_clean
   dh_clean
 debian/rules build
cp foreman_ansible.rb /usr/share/foreman/bundler.d/
cd /usr/share/foreman && ( \
		bundle install --local
/bin/sh: 2: Syntax error: end of file unexpected (expecting ")")
make: *** [build] Error 2
dpkg-buildpackage: error: debian/rules build gave error exit status 2
```

File: plugins/ruby-foreman-ansible/debian/rules [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# As a special exception, when this file is copied by dh-make into a
+# dh-make output file, you may use that output file without restriction.
+# This special exception was added by Craig Small in version 0.37 of dh-make.
+
+# Uncomment this to turn on verbose mode.
+#export DH_VERBOSE=1
+
+PLUGIN = foreman_ansible
+
+build:
```
Comment (domcleal) [inline plugins/ruby-foreman-ansible/debian/rules]: This whole section (and PLUGIN) can be removed if there's no precompilation happening, see ruby-foreman-templates.

File: plugins/ruby-foreman-ansible/debian/changelog [hunk 2]
```diff
@@ -0,0 +1,5 @@
+ruby-foreman-ansible (0.2.1-1) stable; urgency=low
+
+  * Initial release.
+
+ -- Daniel Lobato Garcia <elobatocs@gmail.com>  Sat, 2 Jan 2016 11:09:49 +0200
```
Comment (domcleal) [inline plugins/ruby-foreman-ansible/debian/changelog]: Nitpick usually `02` (padded), but I'm unsure exactly how strict this is.

**Comment by domcleal** [comment]
The `debian/install` file is missing this usual line:

    cache                usr/share/foreman/vendor

It's needed to copy the .gem file into the package so it's installed into Foreman's vendor cache dir.

**Comment by domcleal** [comment]
Merged as 4dc8cb1 to deb/develop and building, thanks @dLobatog.  Should it be in any other releases too?

**Comment by dLobatog** [comment]
It should be fine with 1.8, 1.9, 1.10 if you can cp to these branches? Notice we ought to move the repository to theforeman organization (https://github.com/theforeman/foreman-packaging/pull/947/files#diff-e62ac10b7a24e4818cd9c9c04da1e9d3R7), sounds good @ohadlevy?

**Comment by mmoll** [comment]
cherry-picked and built for 1.8 - 1.10.

**Pull request closed without merging**